pages/form/DaysOfTheWeekSelection.jsx

```diff
@@ -0,0 +1,72 @@
+import { Heading, Box } from "@chakra-ui/react";
+import Layout from "../../components/Layout/Layout";
+import DaysOfTheWeekContainer from "../../components/DaysOfTheWeek/DaysOfTheWeekContainer";
+
+import { BackButton } from "../../components/LinkButton/LinkButton";
+import Q1Progress from "../../public/images/progress-bar/q1-progress-dots.svg";
+import Q1Cloud from "../../public/images/clouds/cloud-q1.svg";
+
+import { useState } from "react";
+import { useRouter } from "next/router";
+import { sendLogs } from "../../utils/sendLogs";
+import useForm from "../../components/FormProvider";
+
+export default function DaysOfTheWeekSelection() {
+
+  /* previous code for question N1 (will be using it for now till we change FormProvider or the data we collect).
+  Even thought the current data inside container component will show exactly which days the user has worked, 
+  we will only save the number of days
+  */
+
+  const { answers, setAnswers } = useForm();
+  const [ nDays, setNDays ] = useState(answers.numDaysWorked);
+
+  const saveAnswers = () => setAnswers(prev => ({ ...prev, numDaysWorked: nDays }));
+
+  const router = useRouter();
+
+  const logMessage = (msg) => {
+    let incentiveMsg = () => {
+      if (!!answers.incentive) {return "<filled>"}
+      else return "<empty>"
+    }
+    return {
+      page: router.pathname,
+      event: msg,
+      ...answers,
+      nWorkDays: nDays,
+      incentive: incentiveMsg(),
+    }
+  }
+
+  // we  pass this function as props to our child component to count humber of days
+  const setNumberOfDays = (days) => {
+    setNDays(days)
+  }
+
+  // we  pass this function as props to our child component to update Form data and logs
+  const saveDataAndShowLog = () => {
+    saveAnswers();
+    sendLogs(logMessage("Next button clicked"));
+  }
+
+  return (
+    <Layout isText={true} Progress={Q1Progress}>
+      <Box pos="absolute" top={["2", "5"]} left={["2", "10"]}>
+        <BackButton
+          href="/"
+          onClick={saveDataAndShowLog}
+        />
+      </Box>
+      <Q1Cloud />
```

Comment:
Probably doesn't matter, but the Cloud image is different from the Figma 😅

Reply:
Thanks @ymloh, it's a good point but for this one it's outside the scope. Cloud is a separate component which is not defined on the page but imported from public folder. It should be changed as a global variable once for all pages.